bluemarket/src/components/Topbar.jsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Topbar.jsx -- Custom Navbar component for the Blue Market App.
+ * TODO: add other pages and link to this page
+ * Components are comprabale to java classes, they can be passed arguments, 
+ * and contain extra javascript functions (ex a function that actually does 
+ * the search when you click the botton). This class does not need arguments to render.
+ *  - it is best practice to have a component in a file with the same name.
+ *  - components must start with capital letters to differentiate from jsx tags.
+ */
+
+// Imports
+import React, { Component } from 'react'; // Import neccesary for all components
+// react-bootstrap was imported to make it easier to make flexible GUI components see https://react-bootstrap.github.io
+import { Navbar, Nav, Form, FormControl, NavDropdown, Button } from 'react-bootstrap';
+export default class Topbar extends Component {
+    /*
+    * Search function to be called when the button is pressed. Will interface 
+    * with the database and grab data from the search bar.
+    */
+    search() {
+        // TODO: Impliment search
```

Comment:
Implement is misspelled in both the TODO and the alert notification

Reply:
fixed